packages/discord-snippet/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "discord-snippet"
+title: "Discord Snippets"
+description: A Collection of snippets for Discord Markdown
+version: 0.1.0
+author: Nathanel Moulin
+tags: ["utility"]
```

Comment:
Would you kindly add a `homepage: ` tag, even if the value is simply `https://github.com/NathanelM/discord-snippet/tree/main`, please? There have been some problems merging packages into the hub when it's not present.